app/src/main/res/menu/toolbar_menu.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/darkModeToggle"
+        android:checkable="true"
+        android:checked="false"
+        app:actionViewClass="android.widget.CheckBox"
+        android:titleCondensed="Night"
+        android:title="Dark Mode" />
+    <item
+        android:id="@+id/notificationToggle"
+        android:checkable="true"
+        android:checked="false"
+        android:title="Notifications" />
```

Comment:
What's the purpose of this menu item?

Reply:
As per our talk on the Twitter, I have mentioned something which I was trying to implement and couldn't figure out so left this part for the future or you can implement too. But I think you haven't noticed that.

Reply:
Okay!